main.py

```diff
@@ -131,6 +136,43 @@ def _evaluate(pipeline_name, dev_mode):
     ctx.channel_send('Recall', 0, recall)
 
 
+def _evaluate_in_chunks(pipeline_name, dev_mode, chunk_size):
```

Comment:
@taraspiotr it seems that there is a lot of repetition between evaluate and predict now. Can we abstract away the  generation of predicted annotations?